hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -811,6 +863,32 @@ func reconcileUserDataSecret(userDataSecret *corev1.Secret, nodePool *hyperv1.No
 	return nil
 }
 
+func reconcileTunedConfigMap(tunedConfigMap *corev1.ConfigMap, nodePool *hyperv1.NodePool, tunedConfig string) error {
+	tunedConfigMap.Immutable = k8sutilspointer.BoolPtr(false)
+	if tunedConfigMap.Annotations == nil {
+		tunedConfigMap.Annotations = make(map[string]string)
+	}
+	if tunedConfigMap.Labels == nil {
+		tunedConfigMap.Labels = make(map[string]string)
+	}
+
+	tunedConfigMap.Labels[tunedConfigMapLabel] = "true"
+	tunedConfigMap.Annotations[nodePoolAnnotation] = client.ObjectKeyFromObject(nodePool).String()
```

Comment:
In addition to the annotation, you could also use a label for the nodepool, and then you can just use the label selector when finding configmaps that belong to the nodepool

Reply:
I may be missing something, but I don't think we can use the full `namespace/name` of NodePools as a Label, as it contains a `/` which is invalid for a label value. 

I assume it's generally safe to assume that NodePool names will be unique across a hosted cluster, but I want to stick with the full namespace/name so that in "phase 2" when NTO is generating a MachineConfig for the NodePool controller to apply, NTO can label/annotate these MachineConfigs with the full namespace/name of the NodePool, so that the NodePool controller knows where to apply them.

Reply:
This configmap ony lives in the control plane namespace, I don't think you need a full namespace/name, but if you wanted to you could just use a different separator. Anyway, it's just to make the querying of configmaps easier. If you don't think it's worth it, I'm ok with that too.

Reply:
I went ahead and made the change you suggested, since NodePool name only (no namespace) is in-line with the https://github.com/openshift/hypershift/pull/1702/files.